CHANGES.md

```diff
@@ -6,6 +6,12 @@ Version 0.3.0
 
 To be released.
 
+ -  The package became to have an assembly for .NET Standard 2.1 besides
+    an existing assembly for .NET Standard 2.0.  The new assembly purposes
+    to support [nullable reference types].
+ -  `Bencodex.Types.Dictionary` became a readonly struct.
+ -  `Bencodex.Types.Dictionary(IEnumerable<KeyValuePair<IKey, IValue>>)`
+    constructor now has no default value for the parameter.
```

Comment:
Shouldn't we leave an id number of this issue, *#24*?

Reply:
@moreal Amended!